tendrl_admin.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
+  tasks:
+    - name: Change admin password to admin123
```

Comment:
Could you note there that this module changes password of user specified in `conf_path` ini file as `usm_username`? Like this:

> Change password of usm admin user (specified as `usm_username` in  `conf_path` ini file) to admin123